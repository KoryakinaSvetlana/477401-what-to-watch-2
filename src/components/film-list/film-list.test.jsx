import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter as Router} from "react-router-dom";
import FilmList from "./film-list";

it(`FilmList correctly render`, () => {
  const tree = renderer.create(
      <Router>
        <FilmList
          films={[{id: 1, name: `1`}, {id: 2, name: `2`}, {id: 3, name: `3`}]}
        />
      </Router>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

