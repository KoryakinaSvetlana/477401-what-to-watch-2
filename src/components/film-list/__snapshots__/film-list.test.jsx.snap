// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after relaunch 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="1"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        1
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="2"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        2
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="3"
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        3
      </a>
    </h3>
  </article>
</div>
`;
